src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
+    private static final Random random = new Random();
```

Comment:
According to the checklist, static methods are generally a bad practice. Instead of using a static variable here, consider creating an instance of Random in the constructor of ColorSupplier.